src/Windowing/OpenToolkit.Windowing.Desktop/OpenToolkit.Windowing.Desktop.csproj

```diff
@@ -1,29 +1,28 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
-    <PropertyGroup>
-      <TargetFrameworks>net472;netcoreapp3.0</TargetFrameworks>
-      <RootNamespace>OpenToolkit.Windowing.Desktop</RootNamespace>
-      <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    </PropertyGroup>
-    
-    <ItemGroup>
-      <ProjectReference Include="..\..\OpenToolkit.Core\OpenToolkit.Core.csproj" />
-      <!-- The PrivateAssets thing is to ensure the .dll.config file gets copied transitively. -->
-      <ProjectReference Include="..\OpenToolkit.GraphicsLibraryFramework\OpenToolkit.GraphicsLibraryFramework.csproj" PrivateAssets="analyzers;build" />
-      <ProjectReference Include="..\..\OpenToolkit.Mathematics\OpenToolkit.Mathematics.csproj" />
-      <ProjectReference Include="..\OpenToolkit.Windowing.Common\OpenToolkit.Windowing.Common.csproj" />
-    </ItemGroup>
-    
-    <ItemGroup>
-      <PackageReference Condition="'$(TargetFramework)' == 'net472'" Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-preview.2">
-        <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-        <PrivateAssets>all</PrivateAssets>
-      </PackageReference>
-      <PackageReference Include="opentoolkit.redist.glfw" Version="3.3.0-pre20190419183326" />
-      <PackageReference Condition="'$(TargetFramework)' == 'net472'" Include="System.Memory" Version="4.5" />
-    </ItemGroup>
-
-    <Import Project="..\..\..\props\common.props" />
-    <Import Project="..\..\..\props\nuget-common.props" />
-    <Import Project="..\..\..\props\stylecop.props" />
-</Project>
+  <PropertyGroup>
+    <TargetFrameworks>net472;netcoreapp3.0</TargetFrameworks>
+    <RootNamespace>OpenToolkit.Windowing.Desktop</RootNamespace>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <ProjectReference Include="..\..\OpenToolkit.Core\OpenToolkit.Core.csproj" />
+    <!-- The PrivateAssets thing is to ensure the .dll.config file gets copied transitively. -->
+    <ProjectReference Include="..\OpenToolkit.GraphicsLibraryFramework\OpenToolkit.GraphicsLibraryFramework.csproj" PrivateAssets="analyzers;build" />
+    <ProjectReference Include="..\..\OpenToolkit.Mathematics\OpenToolkit.Mathematics.csproj" />
+    <ProjectReference Include="..\OpenToolkit.Windowing.Common\OpenToolkit.Windowing.Common.csproj" />
+  </ItemGroup>
+  
+  <ItemGroup>
+    <PackageReference Condition="'$(TargetFramework)' == 'net472'" Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-preview.2">
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>
+    <PackageReference Include="opentoolkit.redist.glfw" Version="3.3.0-pre20190419183326" />
```

Comment:
Are they still `-pre`? If not, please upgrade to the next nonpreview

Reply:
Wasnt added nor removed by this PR, git history is messed up due to auto formatting